topics/imaging/tutorials/yolo_prediction/tutorial.md

```diff
@@ -59,62 +60,88 @@ We will use selected images from the SEANOE dataset:
 
 The SEANOE #101899 collection features real underwater images captured by deep‑sea observatories as part of a citizen science initiative called Deep Sea Spy. These non‑destructive imaging stations continuously monitor marine ecosystems and provide snapshots of various fauna. In this dataset, multiple annotators—including trained scientists and enthusiastic citizen scientists—have manually labeled images with polygons, lines, or points highlighting marine organisms. These annotations are then cleaned and converted into bounding boxes to create a training-ready dataset for object detection with YOLOv8. Though the exact species vary, images often include deep-sea fish, species, making this dataset well-suited for practicing detection tasks.
 
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “Ecoregionalization workflow”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 2. Import images data files and models from [SEANOE marine ,datawarehouse](https://www.seanoe.org/data/00907/101899/)
+>
+>    DeepSeaSpy images data files and models as a zip file
```

Comment:
@yvanlebras for this tutorial, should we use one or two images showcasing species detection from this collection (https://www.seanoe.org/data/00907/101899/) or the full collection is required? thanks!